src/bundle/ApiLoader/BoostFactorProviderFactory.php

```diff
@@ -48,6 +49,10 @@ public function __construct(
 
     public function buildService()
     {
+        if ($this->container === null) {
+            return new BoostFactorProvider();
+        }
```

Comment:
Not having a container set at this point indicates an error. `LogicException` should be preferred in my opinion.

Reply:
Fixed in https://github.com/ibexa/solr/pull/53/commits/4de3f25b223656859e7aa2fd94ddf8631c2d21eb